EIPS/eip-6963.md

```diff
@@ -0,0 +1,230 @@
+---
+eip: 6963
+title: Multi Injected Provider Discovery
+description: Using window events to announce injected providers
+author: Pedro Gomes (@pedrouid), Kosala Hemachandra (@kvhnuke), Richard Moore (@ricmoo), Gregory Markou (@GregTheGreek), Kyle Den Hartog (@kdenhartog), Glitch (@glitch-txs), Sam Wilson (@SamWilsn), Micah Zoltu (@MicahZoltu), Jake Moxey (@jxom), Pierre Bertet (@bpierre), Darryl Yeo (@darrylyeo)
+discussions-to: https://ethereum-magicians.org/t/eip-6963-multi-injected-provider-interface-aka-mipi/14076
+status: Draft
+type: Standards Track
+category: Interface
+created: 2023-05-01
+requires: 1193
+---
+
+## Abstract
+
+A Javascript communication standard that enables multiple wallet providers to be discovered from an EVM-based application using a browser extension injected script through window events.
```

Comment:
This suggestion is actually not accurate since this EIP is NOT an alternative to EIP-1193 since `window.ethereum` is not part of the EIP-1193 spec

In fact this EIP depends on EIP-1193 but instead provides an alternative discovery mechanism to `window.ethereum` which is unwritten industry standard right now

Here is my counter-suggestion:

```suggestion
An alternative discovery mechanism to `window.ethereum` for EIP-1193 providers which supports discovering multiple injected wallet providers in a web page using Javascript's `window` events.
```

Reply:
Works for me!

Reply:
Wallets with no bull don’t use globals

Reply:
`Javascript` → `JavaScript`